src/task_queue/models.py

```diff
@@ -55,10 +57,11 @@ class Task(models.Model):
         to=UserProfile
     )
     last_run = models.DateField(blank=True, null=True)
+    is_archived = models.BooleanField(default=False)
```

Comment:
Do we want this model update?  Doesn't hurt anything but the decision was made not to use it.

Reply:
We might need it in the future, so let's keep it for now.